### DISCUSSION TITLE
remove "continue" button from slider plugin

### DISCUSSION THREAD

**Discussion by n400peanuts** [jspsych/jsPsych#786] [Q&A] [answered]
Hello,

Thank to the excellent support in my last post, the my image-slider-response task is working beautifully!  thanks again!
My issue now is about the continue button: participants simply don't understand that after moving the slider they need to hit the "continue" button in order to record the response, even though there are instructions and practice trials.

I see that this issue has been raised in the past as well in the [google group](https://groups.google.com/forum/#!topic/jspsych/HkdTp97O0D0) but left without answer, so probably is a common problem. How can I record the response of the participant without hitting the continue button? 
Is it possible to simply record the last movement of the slider right before a timeout?

**Comment by vijaymarupudi** [comment, marked as answer]
Does the `response_ends_trial` parameter help?

https://www.jspsych.org/plugins/jspsych-image-slider-response/

**Reply by n400peanuts** [reply]
Hi, no. Even though the response_ends_trial is true the trial doens't end and doesn't record anything until the continue button is pressed. Do you have any suggestions?

**Reply by vijaymarupudi** [reply]
Try this change: 
```diff
diff --git a/plugins/jspsych-image-slider-response.js b/plugins/jspsych-image-slider-response.js
index c1fc680..842cb29 100644
--- a/plugins/jspsych-image-slider-response.js
+++ b/plugins/jspsych-image-slider-response.js
@@ -198,7 +198,7 @@ jsPsych.plugins['image-slider-response'] = (function() {
    // save data
    var trialdata = {
    "rt": response.rt,
    -        "response": response.response
    +        "response": response.response || document.querySelector("#jspsych-image-slider-response-response").valueAsNumber
    };

    display_element.innerHTML = '';
```
